src/config/dynamic-theme-fixes.config

```diff
@@ -7234,6 +7234,11 @@ CSS
     background: none !important;
 }
 
+select[id*="sender-select-sender"] {
```

Comment:
Not available to look on proton mail at the moment. But I fixed yesterday a issue with accounts.protonmail.net

By any chance this element is set to a variable that doesn't exist? Int hat case we be better of overriding that variable.